File: SourceCode/C/day031.c
/* Author: Ahmed A. M. ALOMAR
Date: January 31 - 2020
Description:
This program is an example for If Else Statement
This program will let the user chose three numbers and will show you which one is bigger. */


#include <stdio.h>

int main()
{
  // Some variables
  int x;
  int y;
  int z;
  int result;


  // getting input from keyboard
  printf("Enter the first number: \n");
  scanf("%d", &x);
  printf("Enter the second number: \n");
  scanf("%d", &y);
  printf("Enter the third number: \n");
  scanf("%d", &z);

  // The If Statement
  if (x > y && x > z)
  {
    result = x;
  }
  else if (y > x && y > z)
  {
    result = y;
  }
  else
  {
      result = z;
  }

  printf("The bigger number is:  %d \n", result);

  return 0;
}

File: SourceCode/Python/day031.py
# ------------------------------------ #
# Author: Ahmed A. M. ALOMAR
# Date: May 11 - 2020
#
# Description:
# * Example of Break and Continue in loops.
# * The break example: will iterate
#   through the list until it finds the 
#   object then it will just terminate.
# * The continue example: is a program
#   that will only print the positive 
#   numbers skipping the negatives.
# ------------------------------------ #


# The break keyword
numbers = ["one", "two", "three", "four", "five"]

print("Checking the sale list!")

for number in numbers:
  print(number)
  if number == "three":
    break
    
print("End of search!")


# The continue keyword
whole_numbers = [-5, -4, -3, -2, -1, 0, 1, 2, 3, 4, 5]

for number in whole_numbers:
  if number < 0:
    continue
  print(number)
